internal/dinosaur/pkg/services/quota/ams_quota_service.go

```diff
@@ -56,7 +56,7 @@ func (q amsQuotaService) CheckIfQuotaIsDefinedForInstanceType(dinosaur *dbapi.Ce
 func (q amsQuotaService) hasConfiguredQuotaCost(organizationID string, quotaType ocm.DinosaurQuotaType) (bool, error) {
 	quotaCosts, err := q.amsClient.GetQuotaCostsForProduct(organizationID, quotaType.GetResourceName(), quotaType.GetProduct())
 	if err != nil {
-		return false, err
+		return false, fmt.Errorf("checking configured quota cost: %w", err)
```

Comment:
I have the impression that this is more about retrieving a summary of actually consumed quota, not so much about configured quota.

So, to me the message should read more like:
```suggestion
		return false, fmt.Errorf("retrieving quota costs: %w", err)
```

And, WDYT about checking what other information (see parameters of `GetQuotaCostsForProduct()` are missing from the error context and should hence be added to the error message?